site/src/pages/WorkspacePage/WorkspaceReadyPage.tsx

```diff
@@ -100,13 +100,11 @@ export const WorkspaceReadyPage = ({
       ["canceling", "deleting", "pending", "starting", "stopping"].includes(
```

Comment:
It looks like you can skip ephemeral parameters from the "Create workspace" page:

<img width="1100" alt="Screenshot 2023-07-18 at 12 05 12" src="https://github.com/coder/coder/assets/14044910/8042d7ac-5093-472e-9e92-e7159c35b021">

Reply:
Well noticed!